modules/samples/producers/http/src/main/resources/log4j.properties

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
```

Comment:
@mohanvive 
This is already fixed in commit https://github.com/thiliA/product-cep/commit/33aa0ee34db7d81a11c31eaf3bad9eb5136fee74